Dockerfile.arm64v8

```diff
@@ -1,18 +1,18 @@
-FROM node:14
+FROM arm64v8/node:14-alpine3.13
 WORKDIR /app
 COPY package.json .
 COPY package-lock.json .
 COPY scripts scripts
 COPY index.html .
 COPY tailwind.config.js .
 COPY postcss.config.js .
-RUN npm install --silent && ./scripts/build-css.sh
+RUN npm install --silent && ./scripts/build-css-arm64v8.sh
 
 FROM arm64v8/golang:1.16-alpine
+COPY --from=0 /app/dist /app/dist
```

Comment:
Does the order make a difference?

Reply:
It does, I tested this with it on line 14. You'll get `server.go:29:12: pattern dist/index.html: no matching files found` as an error if it's not on that line. During my investigation it appeared that by the time it got to line 14 there wasn't anything in the dist folder for it to copy over so it failed every time. I tried everything I could to work around this by changing source/dest to different potential folders found by looking inside the containers, each time it failed. Once I thew it on line 12 it worked like a charm.

Reply:
That's super interesting! This might be a bug worth reporting to docker.

Happy to merge this though.